.codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+    # wait for unit and integration test builds.
+    after_n_builds: 2
+  strict_yaml_branch: main  # only use the latest copy on main branch
+
+coverage:
+  precision: 2
+  round: down
+  range: "80...100"
+  status:
+    project:
+      default:
+        enabled: yes
+        target: 85%
```

Comment:
Do we hit this number?

Reply:
No we don't. We can change this number later if it's causing too much noise, or feel free to suggest something more appropriate 👍

Reply:
I'd start with something more realistic which we about to hit, but it's not a blocker

Reply:
I think I'll merge as-is for now and if it causes too much noise we can lower the number.